data_model/1.3/clusters/ResourceMonitoring.xml

```diff
@@ -103,13 +103,13 @@ Davis, CA 95616, USA
       <item value="0" name="UPC" summary="12-digit Universal Product Code">
         <mandatoryConform/>
       </item>
-      <item value="1" name="GTIN" summary="8-digit Global Trade Item Number">
+      <item value="1" name="GTIN-8" summary="8-digit Global Trade Item Number">
```

Comment:
Will check how this is in the SDK and see if we should align.

Reply:
https://github.com/CHIP-Specifications/connectedhomeip-spec/issues/10079